### ISSUE TITLE
New cards does not increase past 5

### ISSUE THREAD

**Issue by bee-san** [lune-stone/anki-addon-limit-new-by-young#24]
Hi! Love the idea.

I have been using this for about a week now but my new cards cannot increase past 5.

Here's how many young cards I have:

![image](https://github.com/lune-stone/anki-addon-limit-new-by-young/assets/10378052/497d8e1d-7c29-4d11-82f3-e3f23254502d)

Here's my config:

```
{
    "limits": [
        {
            "deckNames": "Default::Mining",
            "youngCardLimit": 150
        },
        {
            "deckNames": "Default::Kaishi 1.5k",
            "youngCardLimit": 150
        }
    ],
    "rememberLastUiSettings": true,
    "showNotifications": false,
    "updateLimitsAfterSync": true,
    "updateLimitsIntervalTimeInMinutes": 15,
    "updateLimitsOnApplicationStartup": true,
    "updateLimitsOnInterval": false,
    "utilizationReport": {
        "complete": true,
        "detailLevel": "Verbose",
        "empty": true,
        "noLimit": true,
        "notStarted": true,
        "overLimit": true,
        "subDeck": true,
        "underLimit": true
    }
}
```

So I would expect 73 new cards 

Instead I get:

![image](https://github.com/lune-stone/anki-addon-limit-new-by-young/assets/10378052/e056b55a-fbeb-42d9-bbdf-25e2ab8e9a14)

What's up with this? :(

**Comment by aleksejrs** [comment]
The daily limit is still used https://docs.ankiweb.net/deck-options.html#new-cardsday

**Comment by lune-stone** [comment]
The add-on config looks okay to me. My guess is that the bottle neck is coming from Anki's new-card limit. Double check that the config for the deck under `Options > Daily Limits > Preset > New cards/day` for both the deck and the parent deck to make sure it is large enough (if you want the add-on to be the only limit you can use a large value like 9999).

Once that is done if you want to test it you will likely need to use the `Tools > Limit New by Young > Recalculate today's new card limit for all decks` to see the effect, as the add-on by default will only try to set the limit for the day a single time, and won't update it unless you use the "recalculate" action from the menu. Alternatively waiting until the next day would also works.

Let me know how that goes.

**Comment by bee-san** [comment]
> The add-on config looks okay to me. My guess is that the bottle neck is coming from Anki's new-card limit. Double check that the config for the deck under `Options > Daily Limits > Preset > New cards/day` for both the deck and the parent deck to make sure it is large enough (if you want the add-on to be the only limit you can use a large value like 9999).
> 
> Once that is done if you want to test it you will likely need to use the `Tools > Limit New by Young > Recalculate today's new card limit for all decks` to see the effect, as the add-on by default will only try to set the limit for the day a single time, and won't update it unless you use the "recalculate" action from the menu. Alternatively waiting until the next day would also works.
> 
> Let me know how that goes.

AMAZING! It's the new card preset. thank you so much!

Time to do my 100+ new anki cards haha
